### PR TITLE
Update _MSVC_STL_UPDATE value to February 2020

### PR DESCRIPTION
**Pull request by JeanPhilippeKernel**
# Description

# Checklist

Be sure you've read README.md and understand the scope of this repo.

If you're unsure about a box, leave it unchecked. A maintainer will help you.

- [x] Identifiers in product code changes are properly `_Ugly` as per
  https://eel.is/c++draft/lex.name#3.1 or there are no product code changes.
- [ ] The STL builds successfully and all tests have passed (must be manually
  verified by an STL maintainer before automated testing is enabled on GitHub,
  leave this unchecked for initial submission).
- [x] These changes introduce no known ABI breaks (adding members, renaming
  members, adding virtual functions, changing whether a type is an aggregate
  or trivially copyable, etc.).
- [x] These changes were written from scratch using only this repository,
  the C++ Working Draft (including any cited standards), other WG21 papers
  (excluding reference implementations outside of proposed standard wording),
  and LWG issues as reference material. If they were derived from a project
  that's already listed in NOTICE.txt, that's fine, but please mention it.
  If they were derived from any other project (including Boost and libc++,
  which are not yet listed in NOTICE.txt), you *must* mention it here,
  so we can determine whether the license is compatible and what else needs
  to be done.

### CODE REVIEW

**Comment by JeanPhilippeKernel** [comment]
Some issue with build pipeline for x64?

File: stl/src/xtime.cpp [hunk 1]
```diff
@@ -8,10 +8,10 @@
...
 #include <xtimec.h>
 
-#define NSEC_PER_SEC  1000000000L
-#define NSEC_PER_MSEC 1000000L
-#define NSEC_PER_USEC 1000L
-#define MSEC_PER_SEC  1000
+constexpr long NSEC_PER_SEC = 1000000000L;
+constexpr long NSEC_PER_MSEC = 1000000L;
+constexpr long NSEC_PER_USEC = 1000L;
+constexpr int MSEC_PER_SEC = 1000;
```
Comment (CaseyCarter) [inline stl/src/xtime.cpp]: Any commits added to the branch you submitted for this pull request (PR) are automatically added to the PR. This is convenient when you're making changes in response to review comments and you *want* those changes to appear in the PR, but not so much when you're working on other changes that aren't ready to merge or shouldn't be merged as part of the same PR.

**Comment by CaseyCarter** [comment]
> Some issue with build pipeline for x64?

There are a bunch of source code checks that run only in the x64 build; it would be redundant to run equivalent checks in all four builds, so we picked x64. If you peek at the details

image

in this case it's complaining that you didn't clang-format `stl/src/xtime.cpp` after making changes.

File: stl/src/xtime.cpp (hunk 1 above)
Comment (CaseyCarter) [reply stl/src/xtime.cpp]: (If these changes are intended to be part of this PR, your description should say so and provide some rationale.)

Comment (JeanPhilippeKernel) [reply stl/src/xtime.cpp]: oh okay , I see so let me revert then

**Comment by StephanTLavavej** [comment]
Thanks, and congratulations on your first commit! (I’ll merge this to our Microsoft-internal repo later today. Usually we wait for a test run there to complete, but we know that `_MSVC_STL_UPDATE` updates are never risky. 😸)

**Comment by JeanPhilippeKernel** [comment]
Thank you all :) 
on my way to be an active contributors to learn and help

**Pull request merged**